packages/nimble-components/src/text-area/styles.ts

```diff
@@ -25,6 +31,8 @@ export const styles = css`
         color: ${bodyFontColor};
         flex-direction: column;
         vertical-align: top;
+        --ni-private-scrollbar-width: 17px;
```

Comment:
Like Jesse's comment points out I don't think we can make this assumption about scrollbar widths.
If we can't find a reliable alternative maybe we should push back on the design / see if there is an alternative where we can jump through fewer hoops (or maybe just leave off the icon in text areas)

Reply:
Maybe we can instead measure the offsetWidth - clientWidth and set the error image as a background: https://jsbin.com/culujucize/edit?html,js,output

fewer layout / sizing edge cases to deal with as the icon would then be clipped by the bounds of the control since it is actually rendered as the bg of the control

Reply:
I did use offset/clientWidth properties to determine scrollbar width. I'm not sure how I would go from a Nimble icon element to an image url that could be used for the background, though.

Reply:
This was listed as a brainstorming alternative in-case you keep running into issues, not a prescribed recommendation. I think your min-width etc might address the concerns.

> I'm not sure how I would go from a Nimble icon element to an image url

For reference, it's useful to be aware of the concept of data urls: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs

We have the svg data as a string from nimble tokens. It could be straightforwardly encoded as the url:
```ts
`data:image/svg;base64,${btoa(svgImageData)}`
```

but again you might not need it unless you are running into other issues.